actors/builtin/methods.go

```diff
@@ -8,7 +8,7 @@ const (
 	MethodSend        = abi.MethodNum(0)
 	MethodConstructor = abi.MethodNum(1)
 
-	// TODO: remove this once canonical method numbers are finalized
+	// TODO fin: remove this once canonical method numbers are finalized
```

Comment:
what's fin?

Reply:
sorry means can only be done when all else is closed ("end" in french)

Reply:
I don't think it's helpful

Reply:
I don't understand that comment.